libs/dh/metering-point/data-access-route/src/lib/dh-metering-point-data-access-route.store.ts

```diff
@@ -0,0 +1,29 @@
+/**
+ * @license
+ * Copyright 2021 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Injectable } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+import { map, Observable } from 'rxjs';
+import { dhMeteringPointIdParam } from '@energinet-datahub/dh/metering-point/routing';
+
+@Injectable()
+export class DhMeteringPointDataAccessRouteStore {
+  meteringPointId$: Observable<string> = this.route.params.pipe(
+    map((params) => params[dhMeteringPointIdParam])
+  );
+
+  constructor(private route: ActivatedRoute) {}
+}
```

Comment:
suggestion: I've experimented with this and it seems that there's not much point in creating a local router store as it's almost a 1:1 mapping of `ActivatedRoute`: https://github.com/LayZeeDK/ngrx-router-component-store/blob/main/packages/router-component-store/src/lib/local-router-store.ts

My suggestion is to use `ActivatedRoute` directly instead of wrapping it. At least until we have a global router store that all components and component stores can use.

Reply:
Makes sense at this point in time. Later we can move it again since the metering point id should not be displayed directly from the URL but instead used for making an API request.